ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -101,26 +112,49 @@ EOF
     cat - <<EOF >> "${SHARED_DIR}/dev-scripts-additional-config"
 export NODES_FILE="/root/dev-scripts/bm.json"
 export NODES_PLATFORM=baremetal
-export PRO_IF="eth3"
-export INT_IF="eth2"
 export MANAGE_BR_BRIDGE=n
-export CLUSTER_PRO_IF="enp3s0f1"
 export MANAGE_INT_BRIDGE=n
 export ROOT_DISK_NAME="/dev/sda"
+export NUM_WORKERS=2
+EOF
+
+    # We have 2 baremetal cluster types and both need uniq dev-scripts config
+    # TODO: get these details from ofcir
+    if [ "$(cat $CIRFILE | jq -r .type)" == "cluster_moc" ] ; then
+        scp "${SSHOPTS[@]}" "${CLUSTER_PROFILE_DIR}/esi_cloud_yaml" "root@${IP}:esi_cloud_yaml"
+
+        cat - <<EOF >> "${SHARED_DIR}/dev-scripts-additional-config"
+export CLUSTER_NAME=ci-01
+export LOCAL_REGISTRY_DNS_NAME=virthost.ci-01.okd.on.massopen.cloud
+export PRO_IF="eth0.460"
+export INT_IF="eth0.421"
+export CLUSTER_PRO_IF="eno1"
+export BASE_DOMAIN=okd.on.massopen.cloud
+export PROVISIONING_HOST_EXTERNAL_IP=192.168.55.79
+export EXTERNAL_BOOTSTRAP_MAC=52:54:00:B0:07:4F
+export EXTERNAL_SUBNET=192.168.112.0/24
+export NETWORK_CONFIG_FOLDER=/root/dev-scripts/network-configs/vlan-over-prov
+EOF
+    else
+        cat - <<EOF >> "${SHARED_DIR}/dev-scripts-additional-config"
+export PRO_IF="eth3"
+export INT_IF="eth2"
+export CLUSTER_PRO_IF="enp3s0f1"
 export BASE_DOMAIN="ocpci.eng.rdu2.redhat.com"
 export EXTERNAL_SUBNET_V4="10.10.129.0/24"
 export ADDN_DNS="10.38.5.26"
 export PROVISIONING_HOST_EXTERNAL_IP=$IP
-export NUM_WORKERS=2
```

Comment:
What's the reason to remove here the two workers?

Reply:
Its has been moved a few lines up, as it is common to both cluster types ( "cluster_moc" and "cluster" )